internal/monitors/expvar/expvar.go

```diff
@@ -58,12 +53,12 @@ type Monitor struct {
 	metricPathsParts    map[*MetricConfig][]string
 	dimensionPathsParts map[*DimensionConfig][]string
 	allMetricConfigs    []*MetricConfig
-	logger              logrus.FieldLogger
+	logger              log.FieldLogger
```

Comment:
I'm trying to move away from doing the `logrus` -> `log` alias as it just confuses `goimports` too much.

Reply:
Ah. Yeah I was debating which was the intended one. The nice thing about the alias is if we ever switch out logrus we could keep its API and not have to change all the references to logrus but I suppose s/logrus/something-else isn't a huge deal itself.